defaults/main.yml

```diff
@@ -40,4 +40,14 @@ elasticsearch_default_password: changeme
 
 elasticsearch_password: CHANGEME
 elasticsearch_min_heap_size: 2g
-elasticsearch_max_heap_size: 2g
\ No newline at end of file
+elasticsearch_max_heap_size: 2g
+
+# Offline installation
+elasticsearch_offline_install: false
+elasticsearch_baseurl: 
```

Comment:
Should at least use empty double-quote ("") for better clarity that default var is empty string